### ISSUE TITLE
allow to specify git dirty mark

### ISSUE THREAD

**Issue by rutsky** [pirelenito/git-revision-webpack-plugin#8]
Sometimes I want to include is git working tree is dirty or not into version string, i.e. run with `--dirty` flag:

```
$ git describe --always --tags --dirty
0.0.1-2-g51be290-dirty
```

It would be convenient to be able to do so using git-revision-webpack-plugin, e.g.:

```
module.exports = {
  plugins: [
    new GitRevisionPlugin({dirtyMark: true})
  ]
}
```

or `new GitRevisionPlugin({dirtyMark: '-wip'})`.

**Comment by pirelenito** [comment]
I just published `2.4.0` with added support for [custom versionCommand](https://github.com/pirelenito/git-revision-webpack-plugin#versioncommand-describe---always).

I'll close this issue for now... let me know if it doesn't suit your needs

**Comment by rutsky** [comment]
@pirelenito thanks! This is exactly what I wanted.
